cmd/root.go

```diff
@@ -221,12 +193,12 @@ func runUpdatesWithNotifications(filter t.Filter) {
 	notifier.StartNotification()
 	updateParams := t.UpdateParams{
```

Comment:
We should probably remove this object altogether and pass the whole `WatchConfig` instead?